src/Symfony/Component/Intl/Resources/data/languages/meta.php

```diff
@@ -654,630 +663,650 @@
         12 => 'aka',
         13 => 'akk',
         14 => 'akz',
-        15 => 'ale',
```

Comment:
@nicolas-grekas shouldn't we update the code generator to omit keys for those lists, so that it does not change all following lines when added a new value in the middle of the list ?

Reply:
rebased